MC/bin/o2dpg_qc_finalization_workflow.py

```diff
@@ -61,6 +61,8 @@ def add_QC_finalization(taskName, qcConfigPath, needs=None):
   add_QC_finalization('TOFMatchQC', 'json://${O2DPG_ROOT}/MC/config/QC/json/tofMatchedTracks_ITSTPCTOF_TPCTOF_direct_MC.json')
   add_QC_finalization('tofDigitsQC', 'json://${O2DPG_ROOT}/MC/config/QC/json/tofdigits.json')
   add_QC_finalization('TOFMatchWithTRDQC', 'json://${O2DPG_ROOT}/MC/config/QC/json/tofMatchedTracks_AllTypes_direct_MC.json')
+  add_QC_finalization('tofft0PIDQC', 'json://${O2DPG_ROOT}/MC/config/QC/json/pidft0tof.json')
+  add_QC_finalization('tofPIDQC', 'json://${O2DPG_ROOT}/MC/config/QC/json/pidtof.json')
```

Comment:
We should add if/elseif to add only one QC in finalization
if ( FT0 & TOF) tofft0PIDQC
else if(TOF) tofPIDQC

otherwise nothing

Reply:
This was not addresed, was it?

Reply:
It is not really necessary (it can be skipped)